src/main/java/reposense/authorship/FileInfoAnalyzer.java

```diff
@@ -41,7 +44,7 @@
      * Returns null if the file is missing from the local system, or none of the
      * {@code Author} specified in {@code config} contributed to the file in {@code fileInfo}.
      */
-    public static FileResult analyzeFile(RepoConfiguration config, FileInfo fileInfo) {
+    public static FileResult analyzeNonBinaryFile(RepoConfiguration config, FileInfo fileInfo) {
```

Comment:
instead of `NonBinaryFile`, can we name it `TextFile` for simplicity?

Reply:
Do you mean just here, or everywhere else as well?

Reply:
everywhere. A binary file is commonly known as non-text file. Hence a nonbinary can be simplified as textfile.

Reply:
Ok changed.